templates/base.html

```diff
@@ -56,22 +56,87 @@ <h1 class="brand m-0 h4"><a href="/">2021 Georgia Early Vote Ballot Tracker</a><
-          All of this data comes from the <a href="https://elections.sos.ga.gov/Elections/voterabsenteefile.do">publicly
-          available files</a> published by the Georgia Secretary of State. If
-          you are wanting to track your own ballot status, consider utilizing
-          <a href="https://georgia.ballottrax.net/voter/">Georgia's official
-          site for this purpose</a>.
-        </p>
-      </div>
+        <!-- QUESTION: Why does my override for bg-danger not work but btn-danger does? -->
+        <div class="text-center text-white py-3" style="background-color: #e12814;" >
+          <!-- QUESTION: Should we consider the pre-Eday VA countdown ticker here? -->
+          <h1>The Georgia runoff is on January 5th</h1>
+          <h2>This election will determine who controls the U.S. Senate.</h2>
+        </div>
+
+        <div class="container pt-3 mb-3">
+          <div class="card rounded mb-3">
+            <div class="card-body">
+              <h3 class="card-title mt-0">What's happening</h5>
+              <p class="card-text">Runoff elections are all about turnout: we have <strong>XX days</strong> left to make sure every Georgia voter can make their voice heard. And we're behind: by this point in the November election, <strong>{{ stats.returned_general|commafy }} voters</strong> had already voted. But only <strong>{{ stats.returned_special|commafy }} voters</strong> have voted so far in the runoff. Help us catch up!</p>
+
+              <!-- Left in as placeholder -->
+              <p class='mb-0'>
+                So far, <strong>{{ stats.applied_special|commafy }}</strong>
+                ballots have been requested for the January special election and <strong>{{
+                stats.returned_special|commafy }} ballots</strong> have been returned. There were <strong>{{
+                stats.returned_general|commafy }} ballots </strong> returned early in the general
+                election, so we are only
+                <strong>{{ (100.0 * stats.returned_special / stats.returned_general)|round(1) }}%</strong> of the way there.
+                <em>(As of {{ stats.update_time.strftime('%B %d at %-I:%M %p EST') }}.)</em>
+              </p>
+            </div>
+          </div>
+
+          <div>
+            <h3 class='text-center'>Do your part: Help your friends in Georgia vote</h3>
+            <hr class='mb-4'>
+          </div>
+
+          <div>
+            <h4>1️⃣ Who do you know in Georgia?</h4>
+            <p>Check which of your Facebook friends lives in Georgia.</p>
+            <a href="https://www.facebook.com/search/top?q=my%20friends%20who%20live%20in%20georgia" target="_blank" rel="noopener" class="btn text-white mb-3" style="background-color: #e12814;">Find friends in Georgia on Facebook</a>
+            <p><em>Did you know it's possible to find your friends on Facebook with a text search? Click the button to run the search, we won't post anything to your wall or spam your friends.</em></p>
```

Comment:
What do you think about this language for people who might be nervous about clicking the link?

```suggestion
            <p><em>Did you know it's possible to find your friends on Facebook with a text search? Click the button to run the search, or just go to Facebook and type "my friends in Georgia" into the search box.</em></p>
```

Reply:
👍 seems like a great improvement!